[44_perm_income_cons]_typo

shlff opened the pull request:
Hi @jstac , this PR corrects a possible typo in lecture [perm_income_cons](https://python.quantecon.org/perm_income_cons.html):
- ``So the risk-free loans are being made from one person to another within our closed set of agent.`` -->> ``So the risk-free loans are being made from one person to another within our closed set of agents.``

shlff commented:
Hi @mmcky (cc: @AakashGfude ), I find there are some successful checks for this and another [PR](https://github.com/QuantEcon/lecture-python/pull/339) I submitted after it.

I checked it out that it is not due to merging conflicts like what I faced last time.

Would you mind having a look at them and helping me solve this issue when you have time? Many thanks!

(I will hold on submitting PRs now.)

mmcky commented:
hey @shlff -- this looks like it is passing CI. It might be because your PR needs to be generated from the latest `master` so a `rebase` merge in the latest master may fix your issue. Please feel free to ping me on Slack or we can Zoom.

shlff commented:
> hey @shlff -- this looks like it is passing CI. It might be because your PR needs to be generated from the latest `master` so a `rebase` or merge in the latest master may fix your issue. Please feel free to ping me on Slack or we can Zoom.

Great thanks for helping me fix this, @mmcky . 

I updated and rebased the ``master`` in this and the other PR. Now, these checks seem to be all successful (still waiting for building PDF)! Great thx again!

jstac commented:
Thanks @shlff and @mmcky
